packages/contracts/contracts/L1/ValidatorManager.sol

```diff
@@ -460,9 +460,10 @@ contract ValidatorManager is ISemver, IValidatorManager {
     /**
      * @inheritdoc IValidatorManager
      */
-    function assertCanSubmitOutput(address validator) public view {
-        if (getStatus(validator) != ValidatorStatus.ACTIVE || inJail(validator))
-            revert ImproperValidatorStatus();
+    function assertCanSubmitOutput(address validator) public view returns (bool) {
+        if (getStatus(validator) != ValidatorStatus.ACTIVE) revert ImproperValidatorStatus();
+
+        return true;
```

Comment:
To call this function at etherscan.

Reply:
How about making this to return false if the status is not `ACTIVE` and make the function name to `CanSubmitOutput`? Then the experience at etherscan would be better, and also the code can be simpler at `assertCanSubmitOutput` function at `l2_output_submitter.go`. How do you think?

Reply:
And we can rename this function to `isActivated`.

Reply:
I think it's good idea to change the contract function to `isActivated`. But in the case of `l2_output_submitter.go`, I think it would be helpful to log the validator status as is when the validator is not activated. wdyt?

Reply:
Yes, let's log the status of validator in `l2_output_submitter.go`.